### PR TITLE
update toxcore

### PR DESCRIPTION
**Pull request by quininer**
fix bad tox-bootstrapd.service

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Contact
 If you need help or you would like to submit patches, please seek the
 appropriate channel below.
 
-IRC Channel:          #tox-dev @ chat.freenode.net
-Mailing list:         https://lists.tox.im/listinfo/development
-Github Issue tracker: https://github.com/Tox/arch-repo-tox/issues
+* IRC Channel:          #tox-dev @ chat.freenode.net
+* Mailing list:         https://lists.tox.chat/listinfo/development
```
Comment (Zer0-One) [inline README.md]: this list is just "dev@lists.tox.chat" now

Comment (quininer) [inline README.md]: thx

**Comment by zetok** [comment]
Is there a reason for renaming `tox-git` → `toxcore-git` ?

**Comment by quininer** [comment]
@zetok 
~~In order to avoid duplication and `python-tox`.~~

**Comment by zetok** [comment]
> In order to avoid duplication

Duplication with what?

From what I see, there is no other package named `tox` in arch, also, it doesn't cause conflicts with `python-tox`..?

**Comment by quininer** [comment]
`python-tox` have a `/usr/bin/tox`, `tox-git` named easy to misunderstand a correlation between them.

**Comment by zetok** [comment]
I see.

Can you fix this PR to use `git mv` instead of deleting and adding files?

**Comment by haavard** [comment]
[community] now has a toxcore package, so it makes sense to rename tox-git to toxcore-git.

**Comment by GrayHatter** [comment]
why not libtoxcore? after all it's just a library?

**Comment by haavard** [comment]
libtoxcore-git would be just as fine as toxcore-git imo. However, at this point the package is available in the official repos, and it would be confusing if we did not use the same package name for the -git package.

**Comment by quininer** [comment]
@GrayHatter Because it is not only a library, it also provides a `tox-bootstrapd`.

**Pull request merged**